doc/source/theory/multiphases/cfd_dem/dem.rst

```diff
@@ -1,6 +1,6 @@
-=============================
-Discrete Element Method (DEM)
-=============================
+====================================
+Discrete Element Method (DEM) Theory
+====================================
 
 In this guide, we summarize the theory behind DEM. For further details, we refer the reader to the article by Golshan *et al.* `[1] <https://doi.org/10.1007/s40571-022-00478-6>`_
```

Comment:
We could also provide citation to this one, which I think is easier to read:  https://onlinelibrary.wiley.com/doi/full/10.1002/cjce.23501